syntax = "proto3";

package bank.customer;

message SignUpRequest {
	string userID = 1;
	string password = 2;
	string name = 3;
}

message SignUpResponse {
	bool status = 1;
	string msg = 2;
	string userID = 3;
	string name = 4;
}

message LogInRequest {
	string userID = 1;
	string password = 2;
}

message LogInResponse {
	bool status = 1;
	string msg = 2;
	string userId = 3;
	string name = 4;
}